src/content/dataSubmissions/SubmittedData.tsx

```diff
@@ -152,20 +230,27 @@ const SubmittedData: FC<Props> = ({ submissionId, submissionName }) => {
       return;
     }
 
-    if (error || !d?.getSubmissionNodes || !d?.getSubmissionNodes?.properties?.length) {
+    if (error || !d?.getSubmissionNodes || !("properties" in d.getSubmissionNodes)) {
```

Comment:
Optional alternative, this is the same as `d.getSubmissionNodes?.includes("properties")`